specs/app.spec.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs'),
```

Comment:
Ok for your first comment, as you can see it is displayed incorrectly because of the markdown. When you want to use backticks (``` ` ```) inside markdown, you should encapsulate the whole code between 3 backticks.